src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+export const categorieStatus = () => ({
```

Comment:
- [OPTIONAL  ] Great job but I recommend here to add checkstatus action with duck pattern so kindly add it since it is important when dispatching the action so you don't need to put it in single quotes so make  like this

> const CHECK_STATUS = bookstore/categories/CHECK_STATUS